File: 1-Guess Numbers-猜数字/1-Guess Numbers.java
class Solution {
    public int game(int[] guess, int[] answer) {
        // return (guess[0] == answer[0]?1:0)+(guess[1] == answer[1]?1:0)+(guess[2] == answer[2]?1:0);
        int count = 0;
        for(int i = 0; i<3; i++){
          if(guess[i] == answer[i]){
            count++;
          }
        }
        return count;
    }
}

File: 1-Guess Numbers-猜数字/1-Guess Numbers.js
/**
 * @param {number[]} guess
 * @param {number[]} answer
 * @return {number}
 */
var game = function(guess, answer) {
    // let count = 0;
    // for(let i = 0; i<  3; i++){
    //   if(guess[i] === answer[i]){
    //     count++;
    //   }
    // }
    // return count;
    return (guess[0] == answer[0])+(guess[1] == answer[1])+(guess[2] == answer[2]);
};
